mobile/src/defaultViewComponents/defaultViewTable.js

```diff
@@ -24,31 +25,57 @@ export default class DefaultViewTable extends Component {
 
   render() {
     const { currentFilter, changeTableFilter, reportItem, resolveItem } = this.props
-    const data = this.verifyData()
+    const userData = this.verifyData(true)
+    const data = this.verifyData(false)
     return (
       <View>
         <DefaultViewTableHeader currentFilter={currentFilter} changeTableFilter={changeTableFilter} items={Object.values(this.props.items)}/>
-        <FlatList
+        {userData.length ? <FlatList
```

Comment:
What's wrong with having an empty FlatList when there are no elements?

Reply:
This is to ensure proper centering of the help text